aws/lambdas/code/nagware/package.json

```diff
@@ -4,10 +4,23 @@
   "main": "nagware.js",
```

Comment:
@bryan-robitaille Should we change the file extension here to `.ts` ? Probably not important since the local environment appears to be working.